test/e2e/infinispan/smoke_test.go

```diff
@@ -76,6 +77,32 @@ func TestBaseFunctionality(t *testing.T) {
 	}
 }
 
+// Test if the ConfigMap is applying the custom logging configuration
+func TestCustomLogging(t *testing.T) {
+	defer testKube.CleanNamespaceAndLogOnPanic(t, tutils.Namespace)
+
+	// Infinispan cluster definition with extra labels to be propagated to the service and pods
+	replicas := 1
```

Comment:
The default value is 1 and we're only using this in the `WaitForInfinispanPods` check, so less verbose if we remove this variable and just do `WaitForInfinispanPods(1, ...)`